packages/office-ui-fabric-react/src/components/Button/BaseButton.tsx

```diff
@@ -233,20 +234,37 @@ export class BaseButton extends BaseComponent<IBaseButtonProps, IBaseButtonState
       onRenderAriaDescription = this._onRenderAriaDescription,
       onRenderChildren = this._onRenderChildren,
       onRenderMenu = this._onRenderMenu,
-      onRenderMenuIcon = this._onRenderMenuIcon
+      onRenderMenuIcon = this._onRenderMenuIcon,
+      disabled
     } = props;
+    let { keytipProps } = props;
+    if (keytipProps && menuProps) {
+      keytipProps = {
+        ...keytipProps,
+        hasMenu: true
+      };
+    }
 
     const Content = (
-      <Tag { ...buttonProps }>
-        <div className={ this._classNames.flexContainer } >
-          { onRenderIcon(props, this._onRenderIcon) }
-          { this._onRenderTextContents() }
-          { onRenderAriaDescription(props, this._onRenderAriaDescription) }
-          { onRenderChildren(props, this._onRenderChildren) }
-          { !this._isSplitButton && (menuProps || menuIconProps || this.props.onRenderMenuIcon) && onRenderMenuIcon(this.props, this._onRenderMenuIcon) }
-          { this.state.menuProps && !this.state.menuProps.doNotLayer && onRenderMenu(menuProps, this._onRenderMenu) }
-        </div>
-      </Tag>
+      // If we're making a split button, we won't put the keytip here
+      <KeytipData
```

Comment:
If there's cases where we don't want to put a keytip here, we shouldn't always be putting a keytip here, yeah? Can we instead do something like this: `!this._isSplitButton && <keytipData`